docs/ref/models/database-functions.txt

```diff
@@ -222,8 +222,8 @@ returned when this timezone is active will be the same as above except for:
     The ``week`` ``lookup_type`` is calculated based on `ISO-8601
     <https://en.wikipedia.org/wiki/ISO-8601>`_, i.e.,
     a week starts on a Monday. The first week is the one with the majority
-    of the days, i.e., a week that starts on or before Thursday. The value
-    returned is in the range 1 to 52 or 53.
+    of the days, i.e., the week that contains that year's first Thursday.
```

Comment:
I think the part about "a majority of days" could be removed: "The first week of a year is the one that contains the year's first Thursday."

Reply:
Okay, I took your suggestion and used language from the wikipedia article.

Reply:
Thanks @timgraham and @akulakov for the improvements. Looks good to me 👍